doc/source/whatsnew/v0.25.0.rst

```diff
@@ -656,6 +656,7 @@ I/O
 - Bug in :func:`read_csv` not properly interpreting the UTF8 encoded filenames on Windows on Python 3.6+ (:issue:`15086`)
 - Improved performance in :meth:`pandas.read_stata` and :class:`pandas.io.stata.StataReader` when converting columns that have missing values (:issue:`25772`)
 - Bug in :meth:`DataFrame.to_html` where header numbers would ignore display options when rounding (:issue:`17280`)
+- Bug in :func:`read_hdf` where the `start` and `stop` arguments would raise a ``ValueError`` indicating an index size mismatch (:issue:`11188`)
```

Comment:
this is not very informative of what you are actually addressing, can you re-word

Reply:
commit upcoming